lms/templates/dashboard/certificate_info.underscore

```diff
@@ -0,0 +1,111 @@
+                    }) %>
+                </p>
+            <% } %>
+            </p>
+        <% } %>
+
+        <% if (model.cert_status.show_disabled_download_button || model.cert_status.show_download_url || model.cert_status.show_survey_button){ %>
+            <div class="wrapper-message-primary">
+                <ul class="actions actions-primary">
+                    <% if (model.cert_status.show_disabled_download_button){ %>
+                        <li class="action"><span class="disabled">
+                            <%= _.sprintf("Your %(cert_name_short)s is Generating", {cert_name_short:settings.cert_name_short}) %></span></li>
+                    <% } else if (model.cert_status.show_download_url && model.cert_status.show_cert_web_view){ %>
+                        <li class="action action-certificate">
+                            <a class="btn" href="<%- model.cert_status.cert_web_view_url %>" target="_blank"
+                               title="<%- gettext('This link will open the certificate web view') %>">
+                                <%= _.sprintf(gettext("View %(cert_name_short)s"), {cert_name_short:settings.cert_name_short}) %></a></li>
+                    <% } else if (model.cert_status.show_download_url && $.inArray(model.course_mode, ["honor", "audit", "no-id-professional"])) { %>
+                        <li class="action action-certificate">
+                            <a class="btn" href="<%- model.cert_status.download_url %>"
+                               title="<%- gettext('This link will open/download a PDF document') %>">
+                                <%= _.sprintf(gettext("Download %(cert_name_short)s (PDF)"), {cert_name_short:settings.cert_name_short}) %></a></li>
+                    <% } else if (model.cert_status.show_download_url && model.course_mode === 'verified' && model.cert_status.mode === 'honor'){ %>
+                        <li class="action">
+                            <a class="btn" href="<%- model.cert_status.download_url %>"
+                               title="<%- gettext('This link will open/download a PDF document') %>">
+                                <%= _.sprintf(gettext("Download Your %(cert_name_short)s (PDF)"), {cert_name_short:settings.cert_name_short}) %></a>
+                        </li>
+                    <% } else if (model.cert_status.show_download_url && $.inArray(model.course_mode, ["verified", "professional"])) { %>
+                        <li class="action">
+                            <a class="btn" href="<%- model.cert_status.download_url %>"
+                               title="<%= _.sprintf(gettext('This link will open/download a PDF document of your verified %(cert_name_long)s.'), {cert_name_long:settings.cert_name_long}) %>">
+                                <%= _.sprintf(gettext("Download Your ID Verified %(cert_name_short)s (PDF)"), {cert_name_short:settings.cert_name_short}) %></a>
+                        </li>
+                    <% }
```

Comment:
Please move this logic into the View. I think it can be moved to a function that will generate an object containing
```{
    actionClass: 'string',
    disabled: boolean,
    title: 'string'
}
```